app/src/main/java/com/randomappsinc/studentpicker/listpage/ListActivity.java

```diff
@@ -21,6 +21,9 @@
 
 public class ListActivity extends StandardActivity implements ShakeDetector.Listener {
 
+    public static final int EDIT_NAME_LIST_PAGE_POSITION = 1;
```

Comment:
nit: This should be private. Let's hide the implementation.
```suggestion
    private static final int EDIT_NAME_LIST_PAGE_POSITION = 1;
```